src/upnp_cds.h

```diff
@@ -95,10 +97,11 @@ class ContentDirectoryService {
     /// \brief Samsung Extension X_SetBookmark
     /// \param request Incoming ActionRequest.
     ///
-    static void doSamsungBookmark([[maybe_unused]] const std::unique_ptr<ActionRequest>& request);
+    void doSamsungBookmark(const std::unique_ptr<ActionRequest>& request);
 
     std::shared_ptr<Config> config;
     std::shared_ptr<Database> database;
+    std::shared_ptr<ContentManager> content;
```

Comment:
I guess this is from the first version, looks like its not actually needed now?

Reply:
Yes, you are right, this is not necessary any more. Reverted in: https://github.com/gerbera/gerbera/pull/1288/commits/fc6dc8807a616bbf64c663fc8a625cc62c0760d4